ql/src/test/results/clientpositive/llap/join_1to1.q.out

```diff
@@ -597,22 +592,10 @@ NULL	NULL	NULL	10	10010	66
 NULL	NULL	NULL	25	10025	66
 NULL	NULL	NULL	30	10030	88
 NULL	NULL	NULL	35	10035	88
-NULL	NULL	NULL	40	10040	66
 NULL	NULL	NULL	40	10040	88
-NULL	NULL	NULL	40	10040	88
-NULL	NULL	NULL	50	10050	66
-NULL	NULL	NULL	50	10050	66
-NULL	NULL	NULL	50	10050	66
-NULL	NULL	NULL	50	10050	66
-NULL	NULL	NULL	50	10050	88
 NULL	NULL	NULL	50	10050	88
-NULL	NULL	NULL	50	10050	88
-NULL	NULL	NULL	70	10040	88
 NULL	NULL	NULL	70	10040	88
 NULL	NULL	NULL	70	10040	88
```

Comment:
I didn't verify that all the changes in this file are correct. Should I do it or you went over them already?

Reply:
I've just compared all the results in this file against postgres - and the results are now good